GeosCore/sulfate_mod.F90

```diff
@@ -1045,17 +1045,18 @@ SUBROUTINE SRCSF30( Input_Opt, State_Grid, State_Met, State_Chm, TC2, RC )
     AREA = HcoState%Grid%AREA_M2%Val(:,:)
     !AREA3D(:,:,1) = AREA(:,:)
     !AREA3D(:,:,2) = AREA(:,:)
-
+    
+     ! comment out if shut off subgridcoag bc,14/12/23
     ! Define subgrid coagulation timescale (win, 10/28/08)
-    IF ( TRIM(State_Grid%GridRes) == '4.0x5.0' ) THEN
-       TSCALE = 10.*3600.  ! 10 hours
-    ELSE IF ( TRIM(State_Grid%GridRes) == '2.0x2.5' ) THEN
+    !IF ( TRIM(State_Grid%GridRes) == '4.0x5.0' ) THEN
+    !   TSCALE = 10.*3600.  ! 10 hours
+    !ELSE IF ( TRIM(State_Grid%GridRes) == '2.0x2.5' ) THEN
        TSCALE = 5.*3600.
-    ELSE IF ( TRIM(State_Grid%GridRes) == '0.5x0.625' ) THEN
-       TSCALE = 1.*3600.
-    ELSE IF ( TRIM(State_Grid%GridRes) == '0.25x0.3125' ) THEN
-       TSCALE = 0.5*3600.
-    ENDIF
+    !!ELSE IF ( TRIM(State_Grid%GridRes) == '0.5x0.625' ) THEN
+    !   TSCALE = 1.*3600.
+    !ELSE IF ( TRIM(State_Grid%GridRes) == '0.25x0.3125' ) THEN
+    !   TSCALE = 0.5*3600.
+    !ENDIF
```

Comment:
Can the commented-out code be deleted?

Reply:
Hi @yantosca - would it be possible to leave these commented sections in place for now? The idea being that they could serve as a basis if we decide to make future developments to restore functionality of TOMAS' SGCOAG switch by moving the switch and associated parameters into input options?

Reply:
Yes @BettyCroft. We will leave the commented out code for now.